manipmongo/options.go

```diff
@@ -148,6 +150,28 @@ func OptionExplain(explain map[elemental.Identity]map[elemental.Operation]struct
 	}
 }
 
+// OptionTranslateKeysFromSpecifiers can be used to configure the manipulator to lookup the BSON field name for the specified
+// identities from their associated attribute specification.
+//
+// This option is mostly useful in cases where the exposed attribute name is not the same as the field name that is stored
+// in Mongo as sometimes you need to use a short field name to save space.
+func OptionTranslateKeysFromSpecifiers(specs map[elemental.Identity]elemental.AttributeSpecifiable) Option {
+	return func(c *config) {
+
+		if specs == nil {
```

Comment:
not sure we need that check. If it's nil it's a noop. It can make using it simpler

Reply:
It's clearly invalid if someone provides an uninitialized map and risks panics down the road.

Reply:
Recall that reading from a nip map is a 💥